angular/projects/admin/src/app/services/task.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+import { AngularFireDatabase } from '@angular/fire/database';
+import { AppConfigService } from './app-config.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class TaskService {
+
+  constructor(private database: AngularFireDatabase, private appConfigService: AppConfigService) { }
+
+  updateCache(documentSlug: string) {
```

Comment:
Name it `url` or `path` because we don't use the term "slug" in other parts of the code, in the document object the field is called `url` and that's the value we're passing here.

Reply:
Resolve on this commit https://github.com/oddbit/tanam/pull/200/commits/d124651022f4c5df7197da1920d7da145756e716